commander/tracker/txs_tracker.go

```diff
@@ -0,0 +1,71 @@
+package tracker
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/Worldcoin/hubble-commander/eth"
+	"github.com/ethereum/go-ethereum/core/types"
+)
+
+type TxsTracker struct {
+	TxsChan chan *types.Transaction
+	*TxsSender
+}
+
+func NewTxTracker(ethClient *eth.Client, txsChan chan *types.Transaction) *TxsTracker {
+	return &TxsTracker{
+		TxsSender: newTxRequestsSender(ethClient),
+		TxsChan:   txsChan,
+	}
+}
+
+func (t *TxsTracker) StartTracking(ctx context.Context) error {
+	sentTxs := make([]*types.Transaction, 0)
+
+	for {
+		select {
+		case <-ctx.Done():
+			return nil
+		case reqParams := <-t.requestsChan:
+			_, err := t.sendRequest(reqParams)
+			if err != nil {
+				panic(err)
+			}
+			continue
+		case tx := <-t.TxsChan:
+			sentTxs = append(sentTxs, tx)
+			continue
+		default:
+			break
+		}
+
+		if len(sentTxs) == 0 {
+			time.Sleep(time.Millisecond * 100)
```

Comment:
We block receiving sent txs for 100ms what can cause blocking `sendRequest` function. That ultimately can lead to blocking rollupLoop. We should append sent tx to `sentTxs` as fast as we can.